Styling the "steps" section.

sbastn opened the pull request:
This PR fixes the styles section for "steps". Based on #111.

### Before
<img width="989" alt="screen shot 2015-10-08 at 9 20 58 pm" src="https://cloud.githubusercontent.com/assets/80610/10377234/7ef2ccd8-6e02-11e5-8237-90acc0d081ac.png">
### Now
<img width="983" alt="screen shot 2015-10-08 at 9 19 24 pm" src="https://cloud.githubusercontent.com/assets/80610/10377224/72000284-6e02-11e5-98f2-a6d5492f2efe.png">

weppos commented:
@sbastn in several cases the steps have images on them. Just make sure you are happy with the result of images on the light blue background.

sbastn commented:
@weppos just noticed. What do you think? I might have to sleep on this one ;)
<img width="752" alt="screen shot 2015-10-08 at 9 24 50 pm" src="https://cloud.githubusercontent.com/assets/80610/10377359/1f8ef5c2-6e03-11e5-86f5-0962ffccaa93.png">

jcaudle commented:
Should we go even further and add the ability to use a `<steps>` element instead of the `div` with the special class?

weppos commented:
@sbastn honestly I'm not thrilled about it, but I have the feeling you've got the same feeling :smiley:

jcaudle commented:
For what it's worth, I think the screenshot looks fine on the blue background. It even stands out a bit better than against a white background.

weppos commented:
Specifically, what doesn't convince me is the blue text on a light blue background that is almost identical to the link style and is also confusing with the link in the screenshot.

sbastn commented:
I have removed the background, @weppos was right, it can be confusing since it's the same as the notes and added a small padding and borders. Here is how it looks:

### Without image
<img width="763" alt="screen shot 2015-10-09 at 8 43 04 am" src="https://cloud.githubusercontent.com/assets/80610/10387459/cff1398a-6e61-11e5-92ff-ac734c2d4fa1.png">


### With image
<img width="751" alt="screen shot 2015-10-09 at 8 43 13 am" src="https://cloud.githubusercontent.com/assets/80610/10387462/d47cbfba-6e61-11e5-8e2e-9a9115c453d4.png">

sbastn commented:
I am going to merge this because I am annoyed by what we have out there and it's not even 9am.
